src/scheduler/scheduler.html

```diff
@@ -0,0 +1,110 @@
+<h4 class="frame-title">{{'scheduler' | translate}}</h4>
+<motech-list paginated>
+    <motech-list-header>
+        <div class="button-group-primary">
+            <button class="button-add" ng-click="newJob()" ui-sref="scheduler.createOrUpdateJob({ action: 'new' })" type="button">{{'scheduler.scheduleJob' | translate}}</button>
+        </div>
+        <div class="button-group">
+            <button class="button-filter" type="button" uib-btn-checkbox ng-model="viewFilters" ng-click="showFilters()">{{'scheduler.filters' | translate}}</button>
```

Comment:
@jredlarski // Add .motech-sidebar-button to the filter button, It will take care of all the show/hide logic for you -- that way we don't have to reimplement it on each controller...

Something like the following will work, see how in [app/sidebar.directive.js](https://github.com/motech/motech-ui/blob/master/src/app/sidebar.directive.js)
`<button class="icon-filter motech-sidebar-button">{{ 'email.logging.filters' | translate }}</button>`

checkout the [email/logs.html,](https://github.com/motech/motech-ui/blob/master/src/email/logs.html) it will give you a working example